lib/mac-learning.h

```diff
@@ -189,6 +189,11 @@ struct mac_learning {
      * ports_by_ptr is a hash table indexed by the client-provided pointer. */
     struct hmap ports_by_ptr;   /* struct mac_learning_port hmap_nodes. */
     struct heap ports_by_usage; /* struct mac_learning_port heap_nodes. */
+
+#if defined(P4OVS)
+    /* P4 specific bridge ID */
+    uint8_t p4_bridge_id;
```

Comment:
The convention in this data structure is to use an end-of-line comment.

Reply:
Done